src/features/cellnav/js/cellnav.js

```diff
@@ -1017,17 +1017,11 @@
               grid.api.core.on.scrollEnd($scope, function (args) {
                 // Skip if there's no currently-focused cell
                 var lastRowCol = uiGridCtrl.grid.api.cellNav.getFocusedCell();
-                if (lastRowCol === null) {
-                  return;
-                }
 
-                // if not in my container, move on
-                // todo: worry about horiz scroll
-                if (!renderContainerCtrl.colContainer.containsColumn(lastRowCol.col)) {
+                if (lastRowCol === null) {
```

Comment:
thought: `lastRowCol` is not used, I think the whole callback can be removed at this point.

Reply:
Removed the whole block.

Check :heavy_check_mark: